src/mono/mono/mini/interp/transform.c

```diff
@@ -2090,18 +2135,6 @@ interp_handle_intrinsics (TransformData *td, MonoMethod *target_method, MonoClas
 				td->ip += 5;
 				return TRUE;
 			}
-		} else if (!strcmp (tm, "get_Length")) {
-			MonoClassField *length_field = mono_class_get_field_from_name_full (target_method->klass, "_length", NULL);
```

Comment:
This is not needed any more ?

Reply:
`get_Length` was just `ldarg` + `ldfld` which is easily inlined and optimized by interpreter. Moreover, `get_Length` is usually coupled with a `ldloca`, so in conjunction with `ldfld` it will generate even better code than the "intrinsic", due to already existing optimization